pootle/static/js/browser.js

```diff
@@ -269,8 +267,12 @@ const browser = {
       placeholder: gettext('Entire Project'),
       formatResult: formatResource,
       sortResults: removeCtxEntries,
-    }, handleNavDropDownSelectClick, handleBeforeNavDropDownResourceSelect);
-
+    })
+    $(sel.breadcrumbs).on("change", "select",
```

Comment:
We were passing `sel.resource`, not `sel.breadcrumbs`. Is this ok? Perhaps it is and I just falling asleep...